api/config/swagger.py

```diff
@@ -15,7 +15,7 @@
 def schema_view_swagger(request):
     schema = coreapi.Document(
         title='Questioner API',
-        url='https://questioners-two-staging.herokuapp.com/',  # replace with your url
+        url='http://127.0.0.1:8000/',  # replace with your url
```

Comment:
This is supposed to be the heroku link

Reply:
@Bakley thanks for catching that. will change in the next push